CHANGELOG.md

```diff
@@ -2,15 +2,26 @@
 
 ## v1.0.0-rc.2
 
-*August 3rd, 2022*
+*August 8th, 2022*
 
 This is the third release candidate for Hermes v1.0.0 🎉
 
+### General
+
+- Bumped crates to the following versions:
+  - `ibc-relayer-cli`: 1.0.0-rc.2
+  - `ibc-proto`: 0.19.2
```

Comment:
From the `Cargo.lock`Is this supposed to be 0.20.0 ?
https://github.com/informalsystems/ibc-rs/blob/d722c7ec98d46d0f0de4a802f81711af3f9f9f55/Cargo.lock#L1448-L1460

Reply:
Good catch, thanks!